main.go

```diff
@@ -34,20 +34,24 @@ func run(ctx context.Context) (*api.Api, error) {
 
 	server, err := api.New()
 	if err != nil {
-		slog.Error("Bootin api", "error", err)
+		slog.Error("Booting api", "error", err)
 		return nil, err
 	}
 
 	go func() {
 		if err := server.Start(ctx); err != nil && !errors.Is(err, http.ErrServerClosed) {
-			slog.Error("Starting server", "errror", err)
+			slog.Error("Starting server", "error", err)
 			os.Exit(1)
 		}
 	}()
 
 	return server, nil
 }
 
+func configChanged(oldConfig *config.Config) bool {
```

Comment:
I think the idea for detecting the reload was that we only reload the services that their config was changed,
For example if only our log configs are changed we should not reload the echo server.
What do you think? @Iajrdev